genestack_client/files_util.py

```diff
@@ -468,19 +468,6 @@ def get_groups_to_share(self):
         share_utils = ShareUtil(self.connection)
         return share_utils.get_available_sharing_groups()
 
```

Comment:
Since we haven't properly deprecated this, let's improve it thusly:
```suggestion
    def get_group_folder_info(self, group_accession):
        raise NotImplementedError("FilesUtil.get_group_folder_info has been removed in v0.33")
```